src/accelerate/components/List.tsx

```diff
@@ -156,33 +160,22 @@ export default function List( props: Props ) {
 					</table>
 					<div className="table-footer">
 						<div className="pagination">
-							<button
-								disabled={ page < 2 }
-								type="button"
-								onClick={ () => setPage( page - 1 ) }
-							>
-								← { __( 'Prev', 'altis' ) }
-							</button>
-							{ Array( pagination.pages ).fill( null ).map( ( _, p ) => {
-								if ( p + 1 === page ) {
-									return ( <div className="current">{ p + 1 }</div> );
-								}
-								return (
-									<button
-										type="button"
-										onClick={ () => setPage( p + 1 ) }
-									>
-										{ p + 1 }
-									</button>
-								);
-							} ) }
-							<button
-								disabled={ page <= pagination.pages }
-								type="button"
-								onClick={ () => setPage( page + 1 ) }
-							>
-								{ __( 'Next', 'altis' ) } →
-							</button>
+							{ pagination.total > 0 && (
+								<>
+									<Pagination
+										initialPage={1}
+										itemsPerPage={25}
+										onPageСhange={ ( pageNumber ) =>{
+											setCurrentPage( pageNumber );
+											setPage( pageNumber );
+										} }
+										totalItems={pagination.total}
+										pageNeighbours={5}
+										withGoToInput={true}
+									/>
+									<span className="current-page">{ __( 'Page ', 'altis' ) } { page } { __( 'of ', 'altis' ) } { pagination.pages }</span>
```

Comment:
Check you spacing and formatting all throughout here. The dev server should be warning you about some of it but if not it's the spacing of prop values and removing the superfluous brackets around the single parameter in `onPageChange` arrow function that needs updating.